app/lib/frontend/templates/views/shared/layout.mustache

```diff
@@ -4,8 +4,15 @@
 <!DOCTYPE html>
 <html lang="en-us">
 <head>
+  {{#is_experimental}}
+  <script async="async" src="https://www.googletagmanager.com/gtm.js?id=GTM-MX6DBN9"></script>
```

Comment:
Nit: maybe put only the id behind the {{#is_experimental}} condition?

Reply:
It's more than the ID, it's a different script... I don't know if the contents is different, but it is a different script :D